assets/src/scripts/components/hydrograph/domain.js

```diff
@@ -225,6 +225,14 @@ export const getYTickDetails = function (yDomain, parmCd) {
         tickValues = getFullArrayOfAdditionalTickMarks(tickValues, yDomain);
     }
 
+    // When there are too many log scale ticks they will overlap--reduce the number in proportion to the number of ticks
+    // For example, if there are 37 tick marks, every 4 ticks will be used... if there are 31 tick marks, every 3 ticks
+    // will be used.
+    if (isSymlog && tickValues.length > 20 && mediaQuery(config.USWDS_MEDIUM_SCREEN)) {
+        tickValues = tickValues.sort((a, b) => a - b).filter((_, index) => {
+            return !(index % Math.round(tickValues.length/10));
+        });
+    }
```

Comment:
I ran this code locally and it looks like the scale on mobile doesn't adjust nicely so that it goes back to overlapping labels.

Reply:
> I ran this code locally and it looks like the scale on mobile doesn't adjust nicely so that it goes back to overlapping labels.

Weird. I Would have expected the code on lines 237 through 239 to take care of mobile display.... :thinking:

Reply:
Fixed.